### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.3.5 Identify Input Purpose

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#650]
(Chris L)

1. Agree with what Mary Jo raised here, https://github.com/w3c/matf/issues/2 talks to tests and techniques to a degree, where that is not the nature of the MATF's work. I.e. does it apply vs. how to test aspects of charter come in to play.

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
